src/styles/index.scss

```diff
@@ -1,3 +1,10 @@
 body {
   margin: 0;
+  padding: 0;
 }
+
+@import '/src/utils/variables';
```

Comment:
Ensure that the path to the variables file is correct. It should be relative to the current file location.